packages/Python/lldbsuite/test/lang/swift/availability/TestAvailability.py

```diff
@@ -0,0 +1,129 @@
+                              platform.GetOSMinorVersion(),
+                              platform.GetOSUpdateVersion())
+        program = """
+@available(%s %s, *) func f() {}
+
+// ---------------------------------------------------------------------
+// Method context.
+// ---------------------------------------------------------------------
+class C1 {
+  func method() {
+    print("in method") // break here
+  }
+}
+
+C1().method()
+
+// ---------------------------------------------------------------------
+// Generic method context.
+// ---------------------------------------------------------------------
+class C2 {
+  func method<T>(_ t: T) {
+    print("in method") // break here
+  }
+}
+
+C2().method(0)
+
+// ---------------------------------------------------------------------
+// Method in generic class context.
+// ---------------------------------------------------------------------
+class C3<T> {
+  func method() {
+    print("in method") // break here
+  }
+}
+
+C3<Int>().method()
+
+// ---------------------------------------------------------------------
+// Generic method in generic class context.
+// ---------------------------------------------------------------------
+class C4<U> {
+  func method<V>(_ v: V) {
+    print("in method") // break here
+  }
+}
+
+C4<Int>().method(0)
+
+// ---------------------------------------------------------------------
+// Function context.
+// ---------------------------------------------------------------------
+func f1() {
+  print("in function") // break here
+}
+
+f1()
+
+// ---------------------------------------------------------------------
+// Generic function context.
+// ---------------------------------------------------------------------
+func f2<T>(_ t: T) {
+  print("in function") // break here
+}
+
+f2(0)
+
+// ---------------------------------------------------------------------
+// Top-level context.
+// ---------------------------------------------------------------------
+print("in top_level") // break here
+"""
+        num_breapoints = 7
+        
+        with open(self.getBuildArtifact("main.swift"), 'w') as main:
+            main.write(program %(os_name, version))
+
+        self.build()
+        exe = self.getBuildArtifact("a.out")
+
+        # Create a target by the debugger.
+        target = self.dbg.CreateTarget(exe)
+        self.assertTrue(target, VALID_TARGET)
+
+        # Now create a breakpoint in main.c at the source matching
+        # "Set a breakpoint here"
+        breakpoints = target.BreakpointCreateBySourceRegex(
+            'break here', lldb.SBFileSpec("main.swift"))
+        self.assertTrue(breakpoints and
+                        breakpoints.GetNumLocations() >= num_breapoints,
+                        VALID_BREAKPOINT)
+        process = target.LaunchSimple(None, None, os.getcwd())
+        for i in range(num_breapoints):
+            self.expect("p f()", "can call")
+            process.Continue()
```

Comment:
You could do this with:

continue_to_breakpoint(process, breakpoints)

Then you would know you hit your breakpoint.